src/main/java/de/unistuttgart/towercrushbackend/controller/websockets/WebsocketController.java

```diff
@@ -12,73 +12,34 @@
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
 
 @Controller
 @Slf4j
 public class WebsocketController {
 
     @Autowired
     SimpMessagingTemplate simpMessagingTemplate;
-    String destination = "/topic/messages";
-    String gameDestination = "/topic/games/";
-
-    ExecutorService executorService =
-        Executors.newFixedThreadPool(3);
-    Map<String, Future<?>> submittedLobbies = new HashMap<>();
-    @MessageMapping("/start/lobbyName/{lobbyName}")
-    public void startGame(@DestinationVariable String lobbyName) {
-        log.info("Started task");
-        MessageWrapper startMessage = new MessageWrapper("Started game: " + lobbyName, Purpose.CHAT_MESSAGE);
-        simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-            startMessage);
-        submittedLobbies.put(lobbyName, executorService.submit(() -> {
-            while (true) {
-                MessageWrapper idleMessage = new MessageWrapper(LocalDateTime.now()
-                    + ": doing game: " + lobbyName, Purpose.CHAT_MESSAGE);
-                simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-                    idleMessage);
-                Thread.sleep(10000);
-            }
-        }));
+    final String destination = "/topic/messages";
+    final String lobbyDestination = "/topic/lobbies/";
+
+    @MessageMapping("/lobby/{lobby}/team/{team}/player/{player}")
+    public void joinTeam(@DestinationVariable final String lobby, @DestinationVariable final String team, @DestinationVariable final String player) throws JsonProcessingException {
+        log.info("player: " + player + ", joined team: " + team + ", in lobby: " + lobby);
+        final JoinTeamMessage joinTeamMessage = new JoinTeamMessage(team, player);
+        final String jsonString = convertObjectToJson(joinTeamMessage);
+        final MessageWrapper joinTeamMessageWrapped = new MessageWrapper(jsonString, Purpose.JOIN_TEAM_MESSAGE);
+        simpMessagingTemplate.convertAndSend(lobbyDestination + lobby, joinTeamMessageWrapped);
     }
 
-    @MessageMapping("/lobbyName/{lobbyName}/team/{teamName}/player/{playerName}")
-    public void joinTeam(@DestinationVariable String lobbyName, @DestinationVariable String teamName, @DestinationVariable String playerName) throws JsonProcessingException {
-        log.info("join team");
-        JoinTeamMessage joinTeamMessage = new JoinTeamMessage(teamName,playerName);
-        String jsonString = convertObjectToJson(joinTeamMessage);
-        MessageWrapper joinTeamMessageWrapped = new MessageWrapper(jsonString, Purpose.JOIN_TEAM_MESSAGE);
-        simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-            joinTeamMessageWrapped);
+    @MessageMapping("/start/lobby/{lobby}")
+    public void startLobby(@DestinationVariable final String lobby) {
```

Comment:
does that create a new lobby, or does it start a new game in the lobby?
that's not clear from the name, use either `createLobby` or `startGame`-

Reply:
it starts the game, but like in the frontend i got confused if i use game and lobby at different places so i just used lobby everywhere and made it more specific with the extra attribut. but i also understand your concernes, i can change it when we make the actually game and we defined that in a group, if thats ok for you.

Reply:
In general, I am not really a fan of postponing changes as it typically means not implementing them.
But if you are absolutely certain you'll add it later, I'll allow it.

Reply:
yes i am 100%